ResourceManager/GitExtensionsFormBase.cs

```diff
@@ -41,6 +41,21 @@ public GitExtensionsFormBase()
 #endif
         }
 
+        /// <summary>
+        /// Checks if the form wants to handle the key and executes that hotkey
+        /// (without propagating an unhandled key to the base class function as in <cref>ProcessCmdKey</cref>).
+        /// </summary>
+        public virtual bool ProcessHotkey(Keys keyData)
```

Comment:
Similar to `ProcessCmdKey`:
```suggestion
        public virtual bool ProcessHotKey(Keys keyData)
```

Reply:
> Similar to `ProcessCmdKey`:

This comment was too short for me. What is your intention, @RussKie?

Reply:
We should rename these too :)

Reply:
> We should rename these too :)

1. This would break the settings file.
2. (We are neither .NET nor WinForms.)

Reply:
Touche ;)